frontend/src/router.ts

```diff
@@ -31,8 +31,41 @@ import InspectInsight from './views/InspectInsight.vue';
 import RiskMetrics from "@/views/RiskMetrics.vue";
 import NProgress from "nprogress";
 import SlackConfig from './views/SlackConfig.vue';
+import Login from './views/login.vue';
+import Secure from './views/secure.vue';
```

Comment:
probably not secure.vue either, but I am less sure what this is.

Reply:
@abhinavbajpai2012 : Can you get rid of this line in your branch? 
login.vue and 
secure.vue 

are not files we want in the main folder frontend yet.